x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/index.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import globby from 'globby';
+import path from 'path';
+import type { DeploymentAgnosticFtrProviderContext } from '../../../ftr_provider_context';
+
+const cwd = path.join(__dirname);
+const envGrepFiles = process.env.APM_TEST_GREP_FILES as string;
+
+function getGlobPattern() {
+  try {
+    const envGrepFilesParsed = JSON.parse(envGrepFiles as string) as string[];
+    return envGrepFilesParsed.map((pattern) => {
+      return pattern.includes('spec') ? `**/${pattern}**` : `**/${pattern}**.spec.ts`;
+    });
+  } catch (e) {
+    // ignore
+  }
+  return '**/*.spec.ts';
+}
+
+export default function apmApiIntegrationTests({
+  getService,
+  loadTestFile,
+}: DeploymentAgnosticFtrProviderContext) {
+  // DO NOT SKIP
+  // Skipping here will skip the entire apm api test suite
+  // Instead skip (flaky) tests individually
+  // Failing: See https://github.com/elastic/kibana/issues/176948
+  describe('APM API tests', function () {
```

Comment:
How about renaming it to distinguish these tests from the ones running in stateful?

Reply:
1. easily identify which one is failing
2. easily pick the correct test to run in the flaky runner

Reply:
I thought it was like cypress test but you're right, I just checked it.

leave it as is and if there is a need to change we can update it later. Thanks